jbpm-test-coverage/src/test/resources/quartz-db-short-misfire.properties

```diff
@@ -25,7 +25,7 @@ org.quartz.threadPool.threadPriority = 5
 org.quartz.jobStore.misfireThreshold = 3000
 
 org.quartz.jobStore.class=org.quartz.impl.jdbcjobstore.JobStoreCMT
-org.quartz.jobStore.driverDelegateClass=org.quartz.impl.jdbcjobstore.StdJDBCDelegate
+org.quartz.jobStore.driverDelegateClass=org.jbpm.process.core.timer.impl.quartz.DeploymnetsAwareStdJDBCDelegate
```

Comment:
I think tests will not work since there is a typo in Deploym**nets**AwareStdJDBCDelegate.

Reply:
It seems the tests has passed. Do you have any idea why?